src/hooks/useLocalStorage.js

```diff
@@ -1,9 +1,9 @@
 import { useState } from 'react';
 
-export function useLocalStorage(key, initialValue) {
+export function useLocalStorage(initialValue = '') {
```

Comment:
`export function useLocalStorage(initialValue = '')`

Is there a Use Case where the initial value for the local storage is not sent? I didnt think of this :o